### PR TITLE
test: checkbox functionality

### PR DESCRIPTION
**Pull request by xtina-starr**
The type of this PR is: **test**

This PR resolves [PURCHASE-2219] 
This PR adds a test for the inquiry questions checkbox and serves as an example testing this functionality.

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [ ] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-434]: https://artsyproduct.atlassian.net/browse/CX-434

[PURCHASE-2219]: https://artsyproduct.atlassian.net/browse/PURCHASE-2219

### CODE REVIEW

File: src/lib/Scenes/Artwork/Components/CommercialButtons/InquiryModal.tsx [hunk 1]
```diff
@@ -42,14 +42,13 @@ export const InquiryModal: React.FC<InquiryModalProps> = ({ artwork, ...props })
...
     const isShipping = id === InquiryQuestionIDs.Shipping
-
     return (
       <React.Fragment key={inquiryQuestion}>
         <InquiryField>
           <Flex flexDirection="row" justifyContent="space-between">
             <Flex flexDirection="row">
               <Checkbox
-                data-test-id={`checkbox-${inquiryQuestion}`}
+                data-test-id={`checkbox-${id}`}
```
Comment (erikdstock) [inline src/lib/Scenes/Artwork/Components/CommercialButtons/InquiryModal.tsx:51]: I made this change in #4093 as well 🤝

File: src/lib/Scenes/Artwork/Components/CommercialButtons/__tests__/InquiryModal-tests.tsx [hunk 2]
```diff
@@ -25,7 +25,48 @@ const modalProps = {
   toggleVisibility: jest.fn(),
 }
 
-const TestRenderer = () => (
+interface RenderComponentProps {
```
Comment (erikdstock) [inline src/lib/Scenes/Artwork/Components/CommercialButtons/__tests__/InquiryModal-tests.tsx:28]: These changes may also be relevant to @lilyfromseattle 's Error Banner tests.

File: src/lib/Scenes/Artwork/Components/CommercialButtons/__tests__/InquiryModal-tests.tsx [hunk 3]
```diff
@@ -85,20 +116,32 @@ describe("<InquiryModal />", () => {
...
+        type: "selectInquiryQuestion",
+      })
+    })
   })
 
   describe("user can select Shipping", () => {
     it("user selecting shipping exposes the 'Add your location' CTA", () => {
       const wrapper = getWrapper()
-      wrapper.root.findByProps({ "data-test-id": "checkbox-Shipping" }).props.onPress()
+      wrapper.root.findByProps({ "data-test-id": "checkbox-shipping_quote" }).props.onPress()
```
Comment (erikdstock) [inline src/lib/Scenes/Artwork/Components/CommercialButtons/__tests__/InquiryModal-tests.tsx:137]: 🤝

**Review by erikdstock — COMMENTED** [review]
Looks good from here!

**Comment by xtina-starr** [comment]
@erikdstock this PR doesn't conflict too much with yours. Do you have a preference in the order they get merged.

**Review by zephraph — APPROVED** [review]

**Pull request merged**